tests/unit/test_matrix.py

```diff
@@ -798,3 +798,28 @@ def it_knows_its_unweighted_counts(self):
             _CatXCatMatrix(None, None, unweighted_counts).unweighted_counts,
             unweighted_counts,
         )
+
+
+class Describe_CatXMrMatrix(object):
+    """Unit test suite for `cr.cube.matrix._CatXMrMatrix` object."""
+
+    def it_knows_its_unweighted_counts(self):
+        unweighted_counts = np.array(
+            [
+                [  # -- row 0 ------------
+                    [1, 6],  # -- col 0 --
+                    [2, 5],  # -- col 1 --
+                    [3, 4],  # -- col 2 --
+                ],
+                [  # -- row 1 ------------
+                    [4, 3],  # -- col 0 --
+                    [5, 2],  # -- col 1 --
+                    [6, 1],  # -- col 2 --
+                    # --------------------
+                ],
+            ]
+        )
+        np.testing.assert_equal(
```

Comment:
Is there a way that we could progress to using the `assert` syntax instead of the `np.testisng.assert`? Maybe with using `tolist()` if necessary. But it would certainly be more along the lines of where we're going with Crunch. And more readable

Reply:
Yeah, that's a good idea. I wasn't sure at first how best to assert these but I like the `.tolist()` method. It gets the values, shape right and usually works on the type in practice.

I've updated the unit tests in the matrix subpackage. Not all could be changed because `.tolist()` doesn't give you "almost_equal", so we'll need to figure something else out for that. Also it doesn't work on `NaN` results because `np.nan` is not equivalent to `np.nan` (and also not not-equal, interestingly). So we'll have to noodle what to do with those.